pkg/controller/iam/partialpolicy/bindings_test.go

```diff
@@ -28,7 +28,7 @@ import (
 // mockIdentityResolver helps to resolve referenced member identity
 type mockIdentityResolver struct{}
 
-func (t *mockIdentityResolver) Resolve(member iamv1beta1.Member, memberFrom *iamv1beta1.MemberSource, defaultNamespace string) (string, error) {
+func (t *mockIdentityResolver) Resolve(member iamv1beta1.Member, memberFrom *iamv1beta1.MemberSource, _ string) (string, error) {
```

Comment:
This is an example of where it's tricky to drop the parameter names.  I can guess that `_ context.Context` is a ctx.  But I can't guess what `_ string` is.  And because Go interfaces are so implicit, it's as trivial to figure it out as it would be in e.g. Java...

Reply:
yeah agreed on interface implementations for `_ string ` types it's hard to know what the parameters would be.

I had reverted some here:

https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/09c6b365bb1e416e555f86f27011047e449e70dc/operator/pkg/preflight/upgrade_test.go#L43-L59

But missed these. I'll create an issue for the handful of these; 

```
grep -r --include "*.go" --exclude-dir="third_party" --exclude-dir=".build" -nRH "_ string" .
```